modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -16,17 +19,150 @@ What you can view and edit depends on your permissions.
+
+ . To create a child business group, click the *...* menu next to the business group that will be the parent.
+. In the *Add Business Group* dialog, enter a *Business Group name* and *Owner*, then select from these options: +
+* *Owner can create Business Groups*
++
+Users with the Organization Administrator permission can create child business groups in a business group that they own.
+* *Owner can create environments*
++
+Users with the Organization Administrator permission can create environments within their business groups.
+* *Enable CloudHub global deployment*
++
+This option is available only if global deployment is enabled on the parent business group. When global deployment is enabled, the region is auto-populated according to the region you specified.
+* *Static IPs*: This option is available only if the parent business group has static IPs assigned to it. This option enables the use of static IP addresses.
+. Click *Add Business Group*. +
+The new business group appears in the parent business group hierarchy.
+
+[[navigate-business-groups]]
+== Navigate Between Business Groups
+
+When your organization has multiple business groups, you can switch between them in the list of business groups. Switching between business groups changes the list of available CloudHub deployments, APIs, users, and roles settings.
+
+image::switch-suborg.png["A business group and a list of its sub-groups"]
+
+[NOTE]
+--
+When you create a business group, sign out and sign in to Anypoint Platform to see newly-created business groups in the business group navigation menu.
+When you sign in to Anypoint Platform, you return to the business group where you were last active when you signed out of Anypoint Platform.
+--
+
+If you don't have the Organization Administrator permission, you can view only the business groups that you have permissions to view. In the *Organization* tab, your organization tree displays only the business groups to which you belong.
+
+[[control-business-group-access]]
+=== View User and Team Access 
+
+You can view a list of users or teams that have access to a business group or environment and filter by their assigned permissions.
+
+include::partial$include-nav-business-groups.adoc[]
+
+. Select the *Access Overview* tab. 
++
+A list of users in the current business group is shown.
+ 
+.. Select the business group or environment from the dropdown to view a list of users with access to the selected business group or environment.
+.. Select a permission from the dropdown to see a list of users who have the selected permission in the selected business group or environment. 
+. To see teams with access to the selected business group or environment, select *Teams* from the dropdown.
+. To see teams that have specific permissions assigned, select the permission from the dropdown. 
++
+Permissions that are granted directly to users outside of the Teams feature aren't shown. 
+
+=== View Roles Associated With the Business Group
```

Comment:
```suggestion
=== View Roles Associated with the Business Group
```

Reply:
(Global)
I always have to look that up.

Reply:
https://confluence.internal.salesforce.com/spaces/DOCTEAM/pages/143000605/title+capitalization+Style+Guidelines